Source/platform/locale.cpp

```diff
@@ -170,6 +170,8 @@ std::vector<std::string> GetLocales()
 	}
 
 	CFRelease(languages);
+#elif PS4
+    locales.emplace_back("en_US");

```

Comment:
An empty list is valid as a return value from GetLocales(), you can let the default language be loaded without constructing a temporary string.

```suggestion
    // std::locale is unavailable on PS4 so we just let the fallback language get loaded
```

Comment is a guess at what the cause was, feel free to make it more accurate :D

Reply:
Thanks :)